runtime/nls/j9vm/j9vm_hu.nls

```diff
@@ -2143,21 +2143,21 @@ J9NLS_VM_ILLEGAL_THREAD_STATE_UPCALL.system_action=The JVM will throw a IllegalT
 J9NLS_VM_ILLEGAL_THREAD_STATE_UPCALL.user_response=Ensure the specified linker options for downcall are valid.
 # END NON-TRANSLATABLE
 
-J9NLS_VM_CRIU_RESTORE_RESET_VIRTUALTHREAD_FORKJOINPOOL_PARALLELISM_FORKJOINPOOL_CNF=A JVM nem tudta vissza\u00e1ll\u00edtani java.lang.VirtualThread.ForkJoinPool .p\u00e1rhuzamot java.util.concurrent.ForkJoinPool oszt\u00e1ly nem tal\u00e1lhat\u00f3
+J9NLS_VM_CRIU_RESTORE_RESET_VIRTUALTHREAD_FORKJOINPOOL_PARALLELISM_FORKJOINPOOL_CNF=A JVM nem tudta vissza\u00e1ll\u00edtani java.lang.VirtualThread.ForkJoinPool.parallelism java.util.concurrent.ForkJoinPool oszt\u00e1ly nem tal\u00e1lhat\u00f3
```

Comment:
The base file should note that `java.lang.VirtualThread.ForkJoinPool.parallelism` is not translatable (here and the next two entries).

Reply:
Not sure it's needed? It's basically a note to us (the reviewers of the translations), since the automated translation doesn't pay attention to those comments.